packages/@react-aria/landmark/test/useLandmark.test.tsx

```diff
@@ -72,17 +72,24 @@ function toggleBrowserWindow() {
 
 
 describe('LandmarkManager', function () {
-  let offsetWidth, offsetHeight;
+  let realGetComputedStyle = window.getComputedStyle;
 
   beforeAll(function () {
-    offsetWidth = jest.spyOn(window.HTMLElement.prototype, 'clientWidth', 'get').mockImplementation(() => 1000);
-    offsetHeight = jest.spyOn(window.HTMLElement.prototype, 'clientHeight', 'get').mockImplementation(() => 1000);
+    jest.spyOn(window, 'getComputedStyle').mockImplementation((element) => {
+      if (element.attributes.getNamedItem('data-rsp-testid')?.value === 'scrollview') {
+        const sty = realGetComputedStyle(element);
+        sty.width = '1000px';
+        sty.height = '1000px';
+        return sty;
+      } else {
+        return realGetComputedStyle(element);
+      }
+    });
```

Comment:
Bit annoying since this will need to be done by any users would was mocking offsetWidth like we did before. Open to any ideas